src/components/filter-drop-down/FilterDropDown.css

```diff
@@ -0,0 +1,9 @@
+.FilterDropDown {
+  position: absolute;
```

Comment:
Its good to avoid camel case in class names. Use hyphens and underscores

Reply:
but in BEM we do it right?